futures-util/src/stream/futures_unordered/ready_to_run_queue.rs

```diff
@@ -6,39 +6,39 @@ use std::sync::atomic::AtomicPtr;
 use std::sync::atomic::Ordering::{Relaxed, Acquire, Release, AcqRel};
 
 use super::abort::abort;
-use super::node::Node;
+use super::task::Task;
 
 pub(super) enum Dequeue<Fut> {
-    Data(*const Node<Fut>),
+    Data(*const Task<Fut>),
     Empty,
     Inconsistent,
 }
 
 pub(super) struct ReadyToRunQueue<Fut> {
-    // The task using `FuturesUnordered`.
-    pub(super) parent: AtomicWaker,
+    // The waker of the task using `FuturesUnordered`.
+    pub(super) waker: AtomicWaker,
```

Comment:
I've renamed this to `waker` instead of `parent`. I think that makes it clearer. I named it `parent_waker` for a while, but I think `waker` is clear enough because the queue has only a single one.